packages/expo-updates/ios/EXUpdates/AppLoader/AppLoaderTask.swift

```diff
@@ -19,7 +18,10 @@ public protocol AppLoaderTaskDelegate: AnyObject {
    * AppLoaderTask proceed as usual.
    */
   func appLoaderTask(_: AppLoaderTask, didLoadCachedUpdate update: Update) -> Bool
-  func appLoaderTask(_: AppLoaderTask, didStartLoadingUpdate update: Update)
+  func appLoaderTask(_: AppLoaderTask, didStartCheckingForUpdate body: [String: Any])
```

Comment:
```suggestion
  func appLoaderTask(_: AppLoaderTask, didStartCheckingForRemoteUpdate body: [String: Any])
```

(and similar naming changes for new delegate methods below as was suggested on android)